src/AvaTaxError.cs

```diff
@@ -10,13 +10,19 @@ public class AvaTaxError : Exception
         /// <summary>
         /// The raw error message from the client
         /// </summary>
-        public ErrorResult error { get; set; }
+        public ErrorResult error { get; private set; }
```

Comment:
Removed this change as requested in commit 14c7df5ddcb87b2f8978c983720e6c86eb69bd84